buildSrc/src/main/kotlin/Dependency.kt

```diff
@@ -3,10 +3,11 @@
 object Dependency {
 
     object Kotlin {
-        const val stdlib = "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
+        const val version = "1.8.0"
+        const val stdlib = "org.jetbrains.kotlin:kotlin-bom:$version"
```

Comment:
This is not a stdlib now is it? Bom just makes sure all the version of kotlin libs match.